examples/ConfigStoreDemo/appsettings.json

```diff
@@ -14,5 +14,10 @@
     "Language": "English",
     "Messages": "Hello There;Thanks For Using Azure App Configuration",
     "BackgroundColor": "Orange"
-  }
+  },
+
+  "endpoints": [
+    "https://my-store.azconfig.io",
+    "https://my-store-replica.azconfig.io"
```

Comment:
Please don't hard code any actual endpoints.

Reply:
Somebody could own the appconfig instance "my-store". We shouldn't use endpoints that could belong to real users in examples.

Reply:
We didn't have an example checked in when we used connection string, not sure we need to add one now for this.